pkg/builder/cluster/convert/convert.go

```diff
@@ -132,21 +140,32 @@ func containerToGit(git corev1.Container) (*v1alpha1.SourceSpec, error) {
 	if len(git.Args) < 3 {
 		return nil, fmt.Errorf("Unexpectedly few arguments to git source container: %v", git.Args)
 	}
+	name := ""
+	if len(git.Args) == 6 {
+		name = git.Args[5]
+	}
```

Comment:
can you add a comment to what this Args[5] is?

Reply:
There is a lot of to and from conversion confusion in this code. I will add more comments to explain what is happening.